CLI/action_selector.c

```diff
@@ -63,8 +64,12 @@ static int parse_action_selector_action(int *argc, char ***argv, psabpf_action_t
             return EINVAL;
         }
     } else {
-        fprintf(stderr, "specify an action by name is not supported yet\n");
-        return ENOTSUP;
+        uint32_t action_id = psabpf_action_selector_get_action_id_by_name(ctx, **argv);
+        if (action_id == PSABPF_INVALID_ACTION_ID) {
+            fprintf(stderr, "%s: action not found\n", **argv);
+            return EINVAL;
```

Comment:
Any reason to not return `-EINVAL` here, and follow normal C by returning the negative error value?

Reply:
There are at least two reasons:
- In most cases functions return error codes, useful data is returned using pointers. There is no need to distinguish error code from useful data by the sign of returned value.
- Return code can be passed to `strerror` function, so to avoid double negation, positive value is returned.

Note that value from `errno` also is positive.